features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,17 @@
-# put step definitions in this file
\ No newline at end of file
+# put step definitions in this file
```

Comment:
You should get into the habit of removing comments once you are 'done' with a feature and before you submit a PR.